### ISSUE TITLE
Split up css into smaller chunks

### ISSUE THREAD

**Issue by JammingBen** [owncloud/web#10981]
Currently, all our CSS gets merged into one big file because we've set `cssCodeSplit: false` in the vite config. The goal is to set this to `true` though, as it would improv loading times in Web.

There is, however, one issue where some of the chunks won't be loaded, which leads so missing styles. Needs investigation why this happens. Lazy loading routes could help here, since  chunks from lazy loaded things will definitely be loaded: https://github.com/owncloud/web/issues/10980

**Comment by dj4oC** [comment]
Before this can be acted on, the following needs to be resolved:

- The issue itself states investigation is still needed ("Needs investigation why this happens") into why some CSS chunks fail to load when `cssCodeSplit` is enabled — no root cause is identified yet.
- It's explicitly linked as depending on/related to #10980 (lazy-loaded routes), whose status is unconfirmed.

Could someone confirm whether the missing-styles investigation has progressed, and whether #10980 needs to land first?

🤖 Automated by Claude Code

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by DeepDiver1975** [comment]
This repository is being archived and is no longer maintained. Closing this as part of archival. Thank you for your contribution.
